### DISCUSSION TITLE
How do i import Playlist from Spotify ?

### DISCUSSION THREAD

**Discussion by jameswong3388** [codetheweb/muse#876] [Q&A]
**What's happening?**
How do I import Playlist from Spotify?

**Runtime**
I'm running Muse:
- [ ] Directly from the cloned repository
- [x] Inside a Docker container
- [ ] Something else (please elaborate)

**Versions**
- Muse: Latest
- Docker (if applicable): Latest
- OS: Ubuntu Latest
- Node.js: (if running outside the Docker container)
- ffmpeg: (if running outside the Docker container, 4.1 or later is required)

**Comment by codetheweb** [comment]
`/play <spotify-link>` will fetch the Spotify playlist and attempt to convert tracks to YouTube videos for playback.
